Containerfile

```diff
@@ -32,29 +33,36 @@ USER 1001
 
 WORKDIR /opt/app-root/src
 
-# On container environments, always set a thread budget to avoid undesired thread congestion.
-ENV OMP_NUM_THREADS=4
-
-ENV LANG=en_US.UTF-8
-ENV LC_ALL=en_US.UTF-8
-ENV PYTHONIOENCODING=utf-8
-ENV UV_COMPILE_BYTECODE=1 UV_LINK_MODE=copy
-ENV UV_PROJECT_ENVIRONMENT=/opt/app-root
-
-ENV DOCLING_SERVE_ARTIFACTS_PATH=/opt/app-root/src/.cache/docling/models
-
-COPY --chown=1001:0 pyproject.toml uv.lock README.md ./
-
-RUN --mount=type=cache,target=/opt/app-root/src/.cache/uv,uid=1001 \
+ENV \
+    # On container environments, always set a thread budget to avoid undesired thread congestion.
+    OMP_NUM_THREADS=4 \
+    LANG=en_US.UTF-8 \
+    LC_ALL=en_US.UTF-8 \
+    PYTHONIOENCODING=utf-8 \
+    UV_COMPILE_BYTECODE=1 \
+    UV_LINK_MODE=copy \
+    UV_PROJECT_ENVIRONMENT=/opt/app-root \
+    DOCLING_SERVE_ARTIFACTS_PATH=/opt/app-root/src/.cache/docling/models
+
+COPY --chown=1001:0 README.md ./
```

Comment:
I dont know why README is included so i didnt touch it

Reply:
uv sync doesnt need it

Reply:
I think it was needed with `poetry install`

Reply:
Ok ill remove it